src/test/java/org/sagebionetworks/bridge/services/ParticipantFileServiceTest.java

```diff
@@ -152,7 +152,9 @@ public void createParticipantFile() {
         assertNotNull(result.getCreatedOn());
         assertEquals(result.getUploadUrl(), upload);
         assertEquals(result.getAppId(), "api");
-        assertEquals(TestConstants.TIMESTAMP.compareTo(result.getCreatedOn()), 0);
+        assertEquals(result.getCreatedOn().toDateTime(DateTimeZone.UTC), TestConstants.TIMESTAMP);
+        assertEquals(result.getExpiresOn().toDateTime(DateTimeZone.UTC), TestConstants.TIMESTAMP.plusDays(1));
```

Comment:
Optional: Use .getMillis() instead of .toDateTime(timezone).

Reply:
I appreciate the reminder, just made the switch.